src/atoms/fileActions.test.ts

```diff
@@ -0,0 +1,329 @@
+import { act, renderHook } from '@testing-library/react';
+import { createStore, useAtomValue, useSetAtom } from 'jotai';
+import { describe, expect, test } from 'vitest';
+
+import { readFileContent } from '../filesystem';
+import { pendingAsyncSideEffects } from '../utils/test-utils';
+import {
+  activePaneAtom,
+  closeAllFilesAtom,
+  closeFileFromPaneAtom,
+  focusPaneAtom,
+  leftPaneAtom,
+  openFileAtom,
+  rightPaneAtom,
+  selectFileInPaneAtom,
+  splitFileToPaneAtom,
+} from './fileActions';
+import { FileFileEntry, FolderFileEntry } from './types/FileEntry';
+import { PaneId } from './types/PaneGroup';
+
+vi.mock('../filesystem');
```

Comment:
I have never worked with vitest so maybe this is irrelevant, but shouldn't we setup the mocks in a `beforeEach` or at least `beforeAll` and with something to clear mocks in `afterEach`. It seems very dangerous to me to have this `vi.mock` living here.